script/tool/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
-## NEXT
+## 0.1.2
```

Comment:
Do we need to version this now that it's not published separately in the plugin_tools?

Reply:
It's still published for flutter/packages.